pandas/core/missing.py

```diff
@@ -169,7 +169,10 @@ def _interp_limit(invalid, fw_limit, bw_limit):
     # the beginning (see issues #9218 and #10420)
     violate_limit = sorted(start_nans)
 
-    if limit:
+    if limit is not None:
+        if is_integer(limit) and limit < 1:
+            raise ValueError('`limit` keyword argument must be greater '
```

Comment:
make this consistent with the cython:

``raise ValueError('Limit must be non-negative')``

so make the same message (you pick / combine)